packages/utils/src/cli.ts

```diff
@@ -2,7 +2,7 @@ import { dirname, join, normalize, isAbsolute, relative, resolve, extname, parse
 import { existsSync, readFileSync } from 'node:fs';
 import { readJSONSync, writeJSONSync } from 'fs-extra/esm';
 import { compare } from 'compare-versions';
-import findupSync from 'findup-sync';
+import { findUpSync } from 'find-up';
```

Comment:
nit: findupSync

findup is treated as one lower case word.
https://www.npmjs.com/package/findup-sync

Reply:
`findupSync` is from `findup-sync` and I don't see stopAt there.
  while `findUpSync` is form `find-up`